src/main/java/me/azenet/UHPlugin/listeners/UHGameListener.java

```diff
@@ -134,11 +138,26 @@ public void run() {
 			}, 20L * this.p.getConfig().getInt("death.kick.time", 30));
 		}
 		
+		Location l = ev.getEntity().getLocation();
+		//drop golden apple
+		if(p.getConfig().getBoolean("death.goldenApple.drop")) {
+			try {
+				ItemStack skullApple = new ItemStack(Material.GOLDEN_APPLE);
+				ItemMeta skullMetaApple = skullApple.getItemMeta();
+				skullMetaApple.setDisplayName(ChatColor.GREEN + "Pomme d'or");
```

Comment:
Translation needed here. Or simply don't change the name, so Minecraft handles the translations itself (best solution as for me).